Wrappers/Python/cil/io/ZEISSDataReader.py

```diff
@@ -35,55 +36,71 @@
-        :type file_name: os.path or string
-        :param roi: dictionary with roi to load for each axis.
-                {'axis_labels_1': (start, end, step), 
-                 'axis_labels_2': (start, end, step)}
-                axis_labels are definied by ImageGeometry and AcquisitionGeometry dimension labels.
-                e.g. for ImageData to skip files or to change number of files to load, 
-                adjust 'vertical'. For instance, 'vertical': (100, 300)
-                will skip first 100 files and will load 200 files.
-                'axis_label': -1 is a shortcut to load all elements along axis.
-                Start and end can be specified as None which is equivalent 
-                to start = 0 and end = load everything to the end, respectively.
-                Start and end also can be negative using numpy indexing.
-        :type roi: dictionary, default None
+        ``'axis_label': -1`` is a shortcut to load all elements along axis.
+
+        ``start`` and ``end`` can be specified as ``None`` which is equivalent
+        to ``start = 0`` and ``end = load everything to the end``, respectively.
+    '''
+    
+    def __init__(self, file_name=None, roi=None):
+
+        self.file_name = file_name
 
-        '''
         # Set logging level for dxchange reader.py
         logger_dxchange = logging.getLogger(name='dxchange.reader')
         if logger_dxchange is not None:
             logger_dxchange.setLevel(logging.ERROR)
 
         if file_name is not None:
-            self.set_up(file_name = file_name, roi = roi)
+            self.set_up(file_name, roi = roi)
 
 
     def set_up(self, 
                file_name,
                roi = None):
         '''Set up the reader
+
         
-        :param file_name: file name to read
-        :type file_name: os.path or string, default None
-        :param roi: dictionary with roi to load for each axis.
-                {'axis_labels_1': (start, end, step), 
-                 'axis_labels_2': (start, end, step)}
-                axis_labels are definied by ImageGeometry and AcquisitionGeometry dimension labels.
-                e.g. for ImageData to skip files or to change number of files to load, 
-                adjust 'vertical'. For instance, 'vertical': (100, 300)
-                will skip first 100 files and will load 200 files.
-                'axis_label': -1 is a shortcut to load all elements along axis.
-                Start and end can be specified as None which is equivalent 
-                to start = 0 and end = load everything to the end, respectively.
-                Start and end also can be negative using numpy indexing.
-        :type roi: dictionary, default None
+        Parameters
+        ----------
+        file_name: str
+            file name to read
+        roi: dict, default None 
+            dictionary with roi to load for each axis:
+            ``{'axis_labels_1': (start, end, step),'axis_labels_2': (start, end, step)}``.
```

Comment:
Can we update this to list the two accepted inputs, or add to notes. It'll be `{'angle':(...),'vertical':(...),'horizontal':(...)}` for acquisition data, `{'vertical':(...),'horizontal_y':(...),'horizontal_x':(...)}` for image data